internal/nginx-meshctl/commands/deploy.go

```diff
@@ -620,7 +620,7 @@ func startDeploy(k8sClient k8s.Client, deployer *deploy.Deployer, cleanupOnError
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	signalHandler := newDeploySignalHandle(k8sClient, cleanOnSignal, os.Stdout, deployer.Values.Environment)
+	signalHandler := newDeploySignalHandle(k8sClient, cleanOnSignal, os.Stdout)
```

Comment:
If the environment references are being removed, should we also delete the `--environment` flag code?

Reply:
And the helm chart references as well. I know that's outside the scope of this PR, but it's a little weird removing some parts of the code but not all of it.  My concern is that  we remove some of the references and leave dangling references elsewhere in the code that never get removed.

Reply:
This was dead code. The flag is still used, it just wasn't used for this specific code.